Extensions/Effects/JsExtension.js

```diff
@@ -27,127 +27,289 @@ module.exports = {
+      .addIncludeFile("Extensions/Effects/noise-pixi-filter.js");
     const noiseProperties = noiseEffect.getProperties();
     noiseProperties.set(
-      'noise',
-      new gd.PropertyDescriptor(/* defaultValue= */ '0.5')
-        .setLabel(_('Noise intensity (between 0 and 1)'))
-        .setType('number')
+      "noise",
+      new gd.PropertyDescriptor(/* defaultValue= */ "0.5")
+        .setLabel(_("Noise intensity (between 0 and 1)"))
+        .setType("number")
     );
 
     const sepiaEffect = extension
-      .addEffect('Sepia')
-      .setFullName(_('Sepia'))
-      .setDescription(_('Alter the colors to sepia'))
-      .addIncludeFile('Extensions/Effects/sepia-pixi-filter.js');
+      .addEffect("Sepia")
+      .setFullName(_("Sepia"))
+      .setDescription(_("Alter the colors to sepia"))
+      .addIncludeFile("Extensions/Effects/sepia-pixi-filter.js");
     const sepiaProperties = sepiaEffect.getProperties();
     sepiaProperties.set(
-      'opacity',
-      new gd.PropertyDescriptor(/* defaultValue= */ '1')
-        .setLabel(_('Opacity (between 0 and 1)'))
-        .setType('number')
+      "opacity",
+      new gd.PropertyDescriptor(/* defaultValue= */ "1")
+        .setLabel(_("Opacity (between 0 and 1)"))
+        .setType("number")
+    );
+
+    const displacementEffect = extension
+      .addEffect("Displacement")
+      .setFullName(_("Displacement"))
+      .setDescription(_("Add Displacement effect"))
+      .addIncludeFile("Extensions/Effects/displacement-pixi-filter.js");
+    const displacementProperties = displacementEffect.getProperties();
+    displacementProperties.set(
+      "displacementMapTexture",
+      new gd.PropertyDescriptor("")
+        .setType("resource")
+        .addExtraInfo("image")
+        .setLabel(_("Displacement map texture for the effect"))
+    );
+    displacementProperties.set(
+      "scaleX",
+      new gd.PropertyDescriptor(/* defaultValue= */ "20")
+        .setLabel(_("Scale X"))
+        .setType("number")
+    );
+    displacementProperties.set(
+      "scaleY",
+      new gd.PropertyDescriptor(/* defaultValue= */ "20")
+        .setLabel(_("Scale Y"))
+        .setType("number")
+    );
+
+    const colorMapEffect = extension
+      .addEffect("ColorMap")
+      .setFullName(_("Color Map"))
+      .setDescription(_("Add Color Map effect"))
+      .addIncludeFile("Extensions/Effects/color-map-pixi-filter.js")
+      .addIncludeFile("Extensions/Effects/pixi-filters/filter-color-map.js");
+    const colorMapProperties = colorMapEffect.getProperties();
+    colorMapProperties.set(
+      "colorMapTexture",
+      new gd.PropertyDescriptor("")
+        .setType("resource")
+        .addExtraInfo("image")
+        .setLabel(_("Color map texture for the effect"))
+        .setDescription(
+          _(
+            "Color map are like LUT, you can change colors of pixels by modifing a reference color image containing each colours. Orignal map file can be found at [TODO INSERT PATH]"
```

Comment:
I suggest we host the map file on the wiki and put a link. Use markdown (`[original map file](http://url)`) and I'll update the IDE to be able to display markdown with links in descriptions :)

Reply:
Markdown link are not clickable, and create button are a bit offset.
![image](https://user-images.githubusercontent.com/1670670/71450480-4456b480-2763-11ea-914b-a78fd06e226b.png)

Reply:
I just added support for markdown in master ;)